component/component.js

```diff
@@ -291,3 +305,12 @@ export default Ember.Component.extend(NodeDriver, {
   ]
 });
 
+function ValidateIPaddress(ipaddress) 
```

Comment:
lowercase function name maybe and indentation?

Reply:
done, thank you!